miniupnpd/options.c

```diff
@@ -136,11 +136,7 @@ readoptionsfile(const char * fname, int debug_flag)
 	if(!(hfile = fopen(fname, "r")))
 		return -1;
 
-	if(ary_options != NULL)
-	{
-		free(ary_options);
-		num_options = 0;
-	}
+	num_options = 0;
```

Comment:
so there is a leak if ary_options is already allocated

Reply:
No leak, it's freed in `freeoptions()`.